e2e/paths/enterFactorsAffectingParenting_test.js

```diff
@@ -0,0 +1,26 @@
+const config = require('../config.js');
+
+Feature('EnterFactorsAffectingParenting');
+
+Before((I, caseViewPage) => {
+	I.logInAndCreateCase(config.localAuthorityEmail, config.localAuthorityPassword, config.eventSummary, config.eventDescription);
+	I.waitForElement('.tabs', 10);
+	caseViewPage.goToNewActions(config.applicationActions.enterParentingFactors);
+	I.waitForElement('ccd-case-edit-page', 5);
+});
+
+Scenario('Leaving factors affecting parenting section blank in c110a', (I) => {
+	I.continueAndSubmit(config.eventSummary, config.eventDescription);
+	I.see(`updated with event: ${config.applicationActions.enterParentingFactors}`);
+});
+
+Scenario('There is an optional textArea when I answer yes to a question in factors affecting parenting section', (I, enterFactorsAffectingParentingPage) => {
+	enterFactorsAffectingParentingPage.enterYesForAlcoholOrDrugAbuse();
+	I.seeElement('textArea');
+});
+
+Scenario('Filling in factors affecting parenting sections of c110a', (I, enterFactorsAffectingParentingPage) => {
+	enterFactorsAffectingParentingPage.enterFactorsAffectingParenting();
```

Comment:
In the future we should try to minimise number of test cases to cut on the exec time.

Reply:
I agree, I think we just feel uncomfortable not being able to unit test ccd

Reply:
Agree, that approach is far from being ideal :(